packages/marko-web-theme-monorail/graphql/fragments/content-page.js

```diff
@@ -1,165 +1,208 @@
 const gql = require('graphql-tag');
 
-module.exports = gql`
-fragment ContentPageFragment on Content {
-  id
-  name
-  teaser(input: { useFallback: false, maxLength: null })
-  labels
-  body
-  published
-  updated
-  siteContext {
-    path
-    canonicalUrl
-  }
-  company {
-    id
-    name
-    canonicalPath
-    enableRmi
-  }
-  primarySection {
+const factory = ({ useLinkInjectedBody = false } = {}) => {
+  const fragment = gql`
+
+  fragment ContentPageFragment on Content {
     id
     name
-    alias
-    canonicalPath
-    hierarchy {
+    teaser(input: { useFallback: false, maxLength: null })
+    body(input: { useLinkInjectedBody: ${useLinkInjectedBody} })
+    published
+    labels
+    siteContext {
+      path
+      canonicalUrl
+      url
+    }
+    company {
       id
       name
+      siteContext {
+        path
+      }
+      enableRmi
+      primaryImage {
+        id
+        src(input: { options: { auto: "format,compress" } })
+        alt
+        caption
+        credit
+        isLogo
+      }
+    }
+    primarySection {
+      id
+      name
+      fullName
       alias
       canonicalPath
+      hierarchy {
+        id
+        name
+        alias
+        canonicalPath
+        logo {
+          id
+          src(input: { options: { auto: "format,compress" } })
+        }
+      }
     }
-  }
-  primaryImage {
-    id
-    src(input: { useCropRectangle: true, options: { auto: "format,compress" } })
-    cropRectangle {
-      width
-      height
-    }
-    alt
-    caption
-    credit
-    isLogo
-    cropDimensions {
-      aspectRatio
-    }
-    primaryImageDisplay
-  }
-  gating {
-    surveyType
-    surveyId
-  }
-  userRegistration {
-    isCurrentlyRequired
-    accessLevels
-  }
-  ... on ContentVideo {
-    embedCode
-    transcript
-  }
-  ... on ContentPodcast {
-    transcript
-  }
-  ... on ContentNews {
-    source
-    byline
-  }
-  ... on ContentEvent {
-    ends
-    starts
-  }
-  ... on SidebarEnabledInterface {
-    sidebars: sidebarStubs {
-      name
-      body
-      label
+    primaryImage {
+      id
+      src(input: { options: { auto: "format,compress" } })
```

Comment:
Are we intentionally removing the `useCropRectangle` flag  and fields here? Pretty sure this will break cropped images...

Reply:
Same for crop dimensions and primary image display